apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.usecase.ts

```diff
@@ -69,11 +71,10 @@ export class ConstructFrameworkWorkflow {
 
     const stepType = stepTemplate.type;
     const { stepId } = staticStep;
-
     if (!stepId) {
       throw new InternalServerErrorException(`Step id not found for step ${staticStep.stepId}`);
     }
-
+    console.log('staticStep::', staticStep);
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Remove log‏